### PR TITLE
Move tutorials in modeldocs to docssourcetutorials

### PR DESCRIPTION
**Pull request by gvegayon**
Moves tutorials under `./model/docs` to `./docs/source/tutorials/`, closing #232.

### CODE REVIEW

File: docs/source/developer_documentation.rst [hunk 1]
```diff
@@ -1,8 +1,6 @@
 Developer Documentation
 =======================
 
-**Note this document is a work in progress. Contrbitions to all sections are welcome.**
```
Comment (damonbayer) [inline docs/source/developer_documentation.rst:4]: Is this removal intentional?

**Review by damonbayer — COMMENTED** [review]
@gvegayon This looks good! Other tutorial changes will land soon. I think it would be easiest to merge those first and then update this PR.

File: docs/source/developer_documentation.rst (hunk 1 above)
Comment (gvegayon) [reply docs/source/developer_documentation.rst:4]: I think this is inherited from previous versions of the commit used as a baseline for this PR. No strong opinion here :).

**Comment by gvegayon** [comment]
> @gvegayon This looks good! Other tutorial changes will land soon. I think it would be easiest to merge those first and then update this PR.

Awesome! I'll check which PRs may relate to this. Will ask if I can't find them.

**Comment by damonbayer** [comment]
I think it would also make sense to relocate the make commands for the tutorials in this PR.

**Comment by dylanhmorris** [comment]
Now that #256 and #250 have landed, is it time for this one @damonbayer / @gvegayon?

**Comment by dylanhmorris** [comment]
Now that #250 has landed, is it time for this one @damonbayer / @gvegayon?

File: docs/source/developer_documentation.rst (hunk 1 above)
Comment (dylanhmorris) [reply docs/source/developer_documentation.rst:4]: I'd suggest adding it back in.

Comment (dylanhmorris) [inline docs/source/developer_documentation.rst:4]:
```suggestion
**Note this document is a work in progress. Contrbitions to all sections are welcome.**
```

**Review by dylanhmorris — APPROVED** [review]
Looks good and website appears to build and render properly. I would add back the under construction notice for the dev docs @gvegayon

File: docs/source/developer_documentation.rst [hunk 2]
```diff
@@ -1,8 +1,6 @@
 Developer Documentation
 =======================
```
Comment (gvegayon) [inline docs/source/developer_documentation.rst:3]:
```suggestion

**Note this document is a work in progress. Contrbitions to all sections are welcome.**
```

**Comment by gvegayon** [comment]
Should be good to go no

File: docs/source/tutorials/example_with_datasets.qmd
Comment (damonbayer) [inline docs/source/tutorials/example_with_datasets.qmd:1]: This should have been deleted.

**Comment by damonbayer** [comment]
The developer documentation needs to be updated to reflect this change @gvegayon

File: model/docs/.gitignore
Comment (damonbayer) [inline model/docs/.gitignore:1]: This should have been migrated to the new docs folder.
